tests/test_all.py

```diff
@@ -58,38 +58,47 @@ def test_os_release(auto_container):
     """
     assert auto_container.connection.file("/etc/os-release").exists
 
-    for var_name, value in (
+    for var_name, expected_value in (
         ("VERSION_ID", OS_VERSION),
         ("PRETTY_NAME", OS_PRETTY_NAME),
     ):
-        if OS_VERSION == "tumbleweed" and var_name == "VERSION_ID":
-            # on openSUSE Tumbleweed that is the an ever changing snapshot date
-            # just check whether it starts with current year
-            assert (
-                int(
-                    auto_container.connection.run_expect(
-                        [0], f". /etc/os-release && echo ${var_name}"
-                    ).stdout.strip()[:4]
+        if var_name == "VERSION_ID":
+            if OS_VERSION == "basalt":
+                # Current basalt version number
+                expected_value = "0.1"
+            elif OS_VERSION == "tumbleweed":
+                # on openSUSE Tumbleweed that is the an ever changing snapshot date
+                # just check whether it starts with current year
+                assert (
+                    int(
+                        auto_container.connection.run_expect(
+                            [0], f". /etc/os-release && echo ${var_name}"
+                        ).stdout.strip()[:4]
+                    )
+                    == datetime.datetime.now().year
```

Comment:
I'm ok with it, a better approach coulb be to use the `VERSION_ID` (e.g. `VERSION_ID="20230904"`) and ensure, that it's not older than 30 days. This would mean that we can also release on new years eve, if we feel fancy :slightly_smiling_face: 

Just a suggestion, I'm fine with how it is for now.

Reply:
ok, fixed